Mergin/plugin.py

```diff
@@ -461,7 +542,19 @@ def open_project(self):
 
     def clone_remote_project(self):
         user_info = self.mc.user_info()
-        dlg = CloneProjectDialog(username=user_info["username"], user_organisations=user_info.get("organisations", []))
+        try:
+            workspaces = self.mc.workspaces_list()
+        except (URLError, ClientError) as e:
+            ns = self.mc.global_namespace()
+            workspaces = []  # todo: handle bad server response?
+            if ns:
+                workspaces.append(ns)
+            else:
+                workspaces = None
+
+        dlg = CloneProjectDialog(
+            user_info=user_info, workspaces=workspaces, default_workspace=self.project["namespace"]
```

Comment:
is this source or destination workspace?

Reply:
`workspaces` is the list of available workspaces, `default_workspace` is the workspace that we want to be pre-selected in the workspace combobox